mod/wikirate/features/metric_value.feature

```diff
@@ -13,7 +13,7 @@ Feature: note
     And I fill in "pointer_item" with "Death Star" within "form > fieldset.editor > .RIGHT-company"
     And I fill in "pointer_item" with "2015" within "form > fieldset.editor > .RIGHT-year"
     And I fill in "card_subcards__value_content" with "101"
-    And I fill in "card_subcards__Link_content" with "http://example.com"
+    And I fill in "card_subcards__source_subcards_new_source_subcards__Link_content" with "http://example.com"
     And I press "Submit"
```

Comment:
in concept, these cukes are supposed to use user-facing terms (which might mean we should add labels?)